### ISSUE TITLE
Doesn't support max-width

### ISSUE THREAD

**Issue by Mygod** [Polymer/core-image#9]
I want something like this: Auto stretch (if the image is too large) to fit `core-image` to parent's width by setting `max-width: 100%;`.

**Comment by ebidel** [comment]
`<core-image sizing="cover" fit...>` should do it. max-width won't work because you need to give the internal image a dimension. A couple of the demos show good examples. Alternatively, you could use `width: 100%`.

**Comment by Mygod** [comment]
No, I don't want it to fit the whole parent because I have other text going
above and below it.

**Comment by ebidel** [comment]
So something like `<core-image sizing="cover" style="width:100%;height:150px">`?

**Comment by Mygod** [comment]
Can I leave the height unspecified since there will be multiple images?

**Comment by ebidel** [comment]
You should experiment :)

**Comment by Mygod** [comment]
......okay. When I've got time maybe I will.
